front/app/components/admin/PostManager/components/PostPreview/Idea/AdminIdeaEdit.tsx

```diff
@@ -165,14 +162,9 @@ const AdminIdeaEdit = ({
   };
 
   return (
-    <Box>
+    <Box border="1px solid white" borderRadius={theme.borderRadius}>
```

Comment:
Changing the background color was too hard, so I gave this one + the initiative edit in the admin a white border, which helps it look ok imo. Have a look.